src/components/select-field/BaseSelectField.js

```diff
@@ -80,20 +83,26 @@ class BaseSelectField extends React.Component<Props, State> {
             activeItemID: null,
             activeItemIndex: -1,
             isOpen: false,
+            shouldScrollIntoView: false,
         };
     }
 
-    setActiveItem = (index: number) => {
-        this.setState({ activeItemIndex: index });
+    setActiveItem = (index: number, shouldScrollIntoView?: boolean = true) => {
+        this.setState({ activeItemIndex: index, shouldScrollIntoView });
         if (index === -1) {
             this.setActiveItemID(null);
         }
     };
 
     setActiveItemID = (id: ?string) => {
+        const { shouldScrollIntoView } = this.state;
         const itemEl = id ? document.getElementById(id) : null;
-        this.setState({ activeItemID: id });
-        scrollIntoView(itemEl);
+
+        this.setState({ activeItemID: id, shouldScrollIntoView: false });
+
+        if (shouldScrollIntoView) {
```

Comment:
Should this be part of the setState callback?

Reply:
Yeah, I think so. Will update this